code/modules/cm_aliens/XenoStructures.dm

```diff
@@ -94,7 +94,12 @@
 		else
 			playsound(loc, "alien_resin_break", 25)
 
-		health -= (M.melee_damage_upper + 50) //Beef up the damage a bit
+		var/damage_to_structure = M.melee_damage_upper + XENO_DAMAGE_TIER_7
+		// Builders can destroy beefy things in maximum 5 hits
+		if(isxeno_builder(M))
```

Comment:
I wonder, should this check if the xeno is of the same hive as the resin? or maybe an ally?

Reply:
I could add it, but afaik the current mechanics are the same - xenos get a +50 damage against all resin structures, regardless of hive

Reply:
This is fine imo.